src/components/JournalPage/index.jsx

```diff
@@ -1,20 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+class JournalPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props.getPage(this.props.match.params.pageId);
+  }
 
-const Page = props => (
-  <div className="page">
-    <h3>{props.title}</h3>
-  </div>
-);
 
-Page.defaultProps = {
+  render() {
+    return (
+      <div className="page">
+        <h3>{this.props.title}</h3>
+        <div>
+          {
+            this.props.body.map((el) => {
```

Comment:
does this function loop through the elements of the array?

Reply:
was expecting a  loop :)

Reply:
Yep, `map` is just a standard functional `map` that takes every element and modifies it via a function. It returns an array of items instead of a loop where we'd have to continue adding to an array before returning.